src/i18n/setup.js

```diff
@@ -1,12 +1,12 @@
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
-import TranslationsEn from './translations/english.json';
-import TranslationsSp from './translations/spanish.json';
+import contactEn from './translations/english/contactEn.json';
```

Comment:
Can we place these files under a structure like `locales/en/contact.json` instead? This seems to match up more closely with the extraction docs and some guides like this one:
https://medium.com/swlh/internationalization-with-react-i18next-360a452e8809

Reply:
yep, made the change. I made folders for english and spanish translations